pkg/api/api.go

```diff
@@ -437,6 +437,9 @@ func (hs *HTTPServer) registerRoutes() {
 				// POST Live data to be processed according to channel rules.
 				liveRoute.Post("/push/:streamId/:path", hs.LivePushGateway.HandlePath)
 				liveRoute.Get("/channel-rules", routing.Wrap(hs.Live.HandleChannelRulesListHTTP), reqOrgAdmin)
+				liveRoute.Post("/channel-rules", routing.Wrap(hs.Live.HandleChannelRulesPostHTTP), reqOrgAdmin)
+				liveRoute.Put("/channel-rules/:uid", routing.Wrap(hs.Live.HandleChannelRulesPutHTTP), reqOrgAdmin)
```

Comment:
can we use the pattern rather than uid?  pattern needs to be unique and we are currently not returning uid anyway :shrug: 

![image](https://user-images.githubusercontent.com/705951/133644644-76c23cf8-ef25-46d2-b5e6-94d7cd1bae4d.png)

Reply:
Done, also changed to use JSON body for every op.